src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +61,70 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopologyGroup.IPATrust)
+def test_ipa_trusts__authentication_with_default_settings(client: Client, ipa: IPA, trusted: GenericADProvider):
+    """
+    :title: Authenticate IPA and trusted AD users with default settings
+    :setup:
+        1. Create users
+        2. Start SSSD
+    :steps:
+        1. Authenticate users, one extra time using the fully qualified name for the ipa user
+        2. Authenticate users using the wrong password
+    :expectedresults:
+        1. Logins are successful
+        2. Logins are unsuccessful
+    :customerscenario: False
+    """
+    ipa_user = ipa.user("user1").add(password="Secret123").name
+    ipa_user_fqn = f"{ipa_user}@{ipa.domain}"
+    ad_user = trusted.user("user2").add(password="Secret123").name
+    ad_user_fqn = f"{ad_user}@{trusted.domain}"
```

Comment:
For ad, you can use `ad_user_fqn = trusted.fqn("user2")`

I added this `fqn` method to the IPA role here https://github.com/SSSD/sssd-test-framework/pull/119/files but it won't work until merged obviously